src/renderers/shaders/ShaderChunk/uv_pars_fragment.glsl.js

```diff
@@ -1,5 +1,5 @@
 export default /* glsl */`
-#ifdef USE_UV
+#if defined( USE_UV ) || defined( USE_ANISOTROPY )
```

Comment:
I was not sure what the plan was for USE_UV after https://github.com/mrdoob/three.js/pull/25740 and a hotfix was required. 
Do you recommend setting parameters.vertexUvs when anisotropy is preset or create the define either in WebGLProgram or WebGLRenderer. 
Also, there is vAnisotropyMapUv that could be considered or defined cleanly when no normal map is present.

Reply:
@repalash This is not your fault. Thank you for your contribution.

I think `USE_UV` should be properly defined. Where it is best to define it at this point I would leave to @Mugen87 and yourself.

Reply:
> Also, there is vAnisotropyMapUv that could be considered or defined cleanly when no normal map is present.

I would favor this approach (if feasible) since anisotropy would work similar to clear coat then.